### ISSUE TITLE
Resolution Scale Above 100%

### ISSUE THREAD

**Issue by PweSol** [emoose/DLSSTweaks#114]
About a month ago, PureDark made DLSS mod for GTA V. It allows resolution scale to be increased up to 200%, essentially achieving DLDSR results without the extra steps.

I haven't managed to get DLSSTweaks to work well/at all with resolutions above native. Could this be investigated and made a potential feature?

As a side note, I haven't tried that mod so I cannot confirm myself whether or not the scale is controlled purely by DLSS or of it is some other shenanigans.

**Comment by Skay100** [comment]
> GTA V. It allows resolution scale to be increased up to 200%, 

GTA supports this natively with Frame Scalling mode in game advanced settings.

**Comment by PweSol** [comment]
> > GTA V. It allows resolution scale to be increased up to 200%,
> 
> GTA supports this natively with Frame Scalling mode in game advanced settings since it released.

Yes, but I would like to see something similar with DLSSTweaks for any DLSS enabled game.
